README.md

```diff
@@ -70,8 +70,12 @@ The attributes listed below are properly formatted as JSON in [*example.json*](h
 
 >>**text** (string):  Defines the default button text. The default is `"Continue"`.
 
+>>**disabledText** (string):  Defines the default button text when the button is disabled. If not set, `text`, `startText`, or `finalText` will be used. `_styleBeforeCompletion` must be set to `visible`. The default is `""`.
```

Comment:
Please rewrite the latter options in order of precedence, the final paragraph of the pr body needs changing in the same regard also.